pkg/cmd/install.go

```diff
@@ -321,6 +323,26 @@ func (o *installCmdOptions) install(cobraCmd *cobra.Command, _ []string) error {
 				}
 			}
 		}
+
+		if size := len(o.MavenBuildExtension); size > 0 {
+			platform.Spec.Build.Maven.Extension = make([]v1.Extension, 0, size)
+			for _, extension := range o.MavenBuildExtension {
+				gav := strings.Split(extension, ":")
```

Comment:
I think we already have some util methods that parse GAV somewhere.

Reply:
There is a [ParseGav](https://github.com/apache/camel-k/blob/main/pkg/util/maven/maven_command.go#L242) method (if that's what you had in mind ?). I thought about using it but I see two problems :
1. It allows classifiers and packaging types which isn't allowed for extensions
2. The error message isn't quite adapted (due to 1.)

Let me know what you think

Reply:
Ah right, that's the one I had in mind. Not a big deal. Let's go with your proposition 👍🏼.